rubrics/tool/src/main/frontend/webcomponents/js/sakai-rubrics-manager.js

```diff
@@ -104,6 +111,27 @@ class SakaiRubricsManager extends RubricsElement {
       icon.removeClass("fa-chevron-down").addClass("fa-chevron-right");
     }
   }
+
+  filterRubrics() {
```

Comment:
Would array.filter be the ideal way to remove jquery from this? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter